src/audio/audio.cpp

```diff
@@ -144,7 +144,9 @@ void Audio::subscribeInput()
     if (!inputSubscriptions++)
     {
         openInput(Settings::getInstance().getInDev());
-        openOutput(Settings::getInstance().getOutDev());
+        if (!alOutDev)
+            openOutput(Settings::getInstance().getOutDev());
```

Comment:
@tux3 Why do we try to open the output audio device in subscribeInput()? Could you check without? I currently cannot build… :(